previousnext/agov
# [#32862] Themed article teaser dates

Steps to test:

1) Check that date that don't have a label looks as follows(this includes teasers)
Teaser:
![](https://dl.dropbox.com/s/2wrhcj5k2qt477r/Screenshot%202015-07-16%2015.02.49.png?dl=0)

Full content:
![](https://dl.dropbox.com/s/mieuydc3qh48xy7/Screenshot%202015-07-16%2017.06.00.png?dl=0)

2) Check dates with a label in full content pages look as follows e.g publications:
   ![](https://dl.dropbox.com/s/61l5unwueer3zyn/Screenshot%202015-07-16%2017.08.23.png?dl=0)

3) Check that the date in event full content type has a label and is below the intro para
    ![](https://dl.dropbox.com/s/z24hstesbiodkl0/Screenshot%202015-07-16%2017.10.09.png?dl=0)
